Karma-Khameleons/bangazon-website
# Customer can register

## Description
A customer can now register via the website / register link


## Number of Fixes
2


## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#29 #3 #6

## Expected Behavior
A user should be able to go to the website ```localhost:8000``` and be able to register


## Steps to Test Solution
checkout branch ```amCustomerRegister```
makemigrations / migrate (changes were made to Customer)
visit ```localhost:8000``` and click on register
once registered, you should see the products page

## Unit Testing

## Documentation
